src/segment-loader.js

```diff
@@ -1255,40 +1262,45 @@ export default class SegmentLoader extends videojs.EventTarget {
       // we're downloading a single rate (and don't know what that rate is)
       // so we just use the seconds in the buffer to calculate the new buffer
       // sizes.
-      maxForwardBuffer = Math.max((forwardBuffer / configTotalBuffer) * secondsInBuffer,
+      newForwardBufferGoal = Math.max((forwardBufferGoal / totalBufferGoal) * secondsInBuffer,
         this.playlist_.targetDuration);
-      maxReverseBuffer = secondsInBuffer - maxForwardBuffer;
+      newBackBufferGoal = secondsInBuffer - newForwardBufferGoal;
     } else {
       // since we potentially have more than one rate in the buffer, we use
       // the amount of bytes in the buffer to calculate the upper limit on
       // the equivalent time at the current rate
       let upperTimeLimit = (bytesInBuffer * 8) / this.playlist_.attributes.BANDWIDTH;
 
-      maxForwardBuffer = (forwardBuffer / configTotalBuffer) * upperTimeLimit;
-      maxReverseBuffer = (reverseBuffer / configTotalBuffer) * upperTimeLimit;
+      newForwardBufferGoal = (forwardBufferGoal / totalBufferGoal) * upperTimeLimit;
+      newBackBufferGoal = (backBufferGoal / totalBufferGoal) * upperTimeLimit;
     }
 
-    maxForwardBuffer = Math.min(forwardBuffer, maxForwardBuffer);
-    maxReverseBuffer = Math.min(reverseBuffer, maxReverseBuffer);
+    newForwardBufferGoal = Math.min(forwardBufferGoal, newForwardBufferGoal);
+    newBackBufferGoal = Math.min(backBufferGoal, newBackBufferGoal);
 
-    loggerMsg = '\n\tbytes in buffer: ' + bytesInBuffer +
-      ', seconds in buffer: ' + secondsInBuffer +
-      '\n\tforward buffer: ' + forwardBuffer + ' => ' + maxForwardBuffer +
-      '\n\treverse buffer: ' + reverseBuffer + ' => ' + maxReverseBuffer;
+    loggerMsg =
+      `\n\tbytes in buffer: ${bytesInBuffer}, seconds in buffer: ${secondsInBuffer}` +
+      `\n\tforward buffer: ${forwardBufferGoal} => ${newForwardBufferGoal}` +
+      `\n\treverse buffer: ${backBufferGoal} => ${newBackBufferGoal}`;
 
     if (setByteLimit) {
-      loggerMsg += '\n\tbyte limit: ' + this.maxBytes_ + ' => ' + bytesInBuffer;
+      loggerMsg += `\n\tbyte limit: ${this.maxBytes_} => ${bytesInBuffer}`;
```

Comment:
I wonder if this log would be a little inaccurate if we end up staying at the original `maxBytes_` value